### ISSUE TITLE
benchmarks

### ISSUE THREAD

**Issue by antoniomuso** [Brooooooklyn/snappy#54]
Hello, I see in the release `7.1.1` snappy has an increase in performance of 2-3 times. But I did some local tests and I don't note the difference in performance between version `7.0.5` and` 7.1.1`. I note that you have changed the computer for the benchmarks, is it possible you have compared results of different hardware?

**Comment by simonecorsi** [comment]
I'm genuinely curious about this too! I ran some benchmarks and the performance is great tho but I'm not seeing those x2/x3 mentioned in the release notes and was wondering if it was a typo!

**Comment by Brooooooklyn** [comment]
Oh sorry. I forgot the performance improvement codes have been shipped in `7.0.4`. So the performance in `7.1.x` is basically identical with `7.0.4`.

I have benchmarked the latest code aginst the `7.0.3` and here is the result:

```
Running "Compress" suite...
Progress: 100%

  snappy:
    2 438 ops/s, ±3.14%   | fastest

  snappy-7.0.3:
    1 862 ops/s, ±1.51%   | 23.63% slower

  snappy-v6:
    1 340 ops/s, ±1.84%   | slowest, 45.04% slower
```

I'll update the changelog in release notes, thanks.
